### DISCUSSION TITLE
Why is there a limit of 3 languages?

### DISCUSSION THREAD

**Discussion by TheBestPessimist** [scambier/obsidian-text-extractor#57] [Q&A] [answered]
![image](https://github.com/scambier/obsidian-text-extractor/assets/4482210/749ae95b-b83f-49bf-ba71-a85c56f8e657)

**Comment by scambier** [comment, marked as answer]
The first language in the list will have the best accuracy, so the more you add, the less accurate your results will be.
I _could_ remove the limit and let users add 50 languages (because why not), but then I'd have issue reports about incorrect results 🤷‍♂️

**Reply by TheBestPessimist** [reply]
On the one hand i agree, on the other, my (girlfriend's) usecase is searching through PDFs from english, romanian, french, german, hungarian, czech, polish, russian :)).

_PHD's are hard..._

**Reply by scambier** [reply]
PDFs do not go through OCR, they go through a different library that (tries to) extract plain text values. It's totally unrelated to that setting :)

**Reply by TheBestPessimist** [reply]
she has a lot of PDFs which are just pictures -- hundreds of pages of pictures. Those don't get OCR? Then how do we get text from them?

**Reply by scambier** [reply]
> Those don't get OCR?

Nope, sorry :/ Images _in_ PDFs don't get extracted

**Reply by TheBestPessimist** [reply]
Ah, so what the library does regarding PDF is to simply index the text inside?

**Reply by scambier** [reply]
yup

**Reply by TheBestPessimist** [reply]
thank you very much for the explanation
